server/types/oid/iterate.go

```diff
@@ -541,6 +612,15 @@ func runFunction(ctx *sql.Context, oid uint32, callbacks Callbacks) error {
 	return err
 }
 
+// runType is called by RunCallback to handle Section_Function.
+func runType(ctx *sql.Context, toid uint32, callbacks Callbacks) error {
+	_, err := callbacks.Type(ctx, ItemType{
+		OID:          toid,
+		StandardName: getStandardNameFromOid(toid),
+	})
```

Comment:
This doesn't assign the type to `ItemType.Item`. It also doesn't validate that the given OID is a valid type OID. Easiest way to implement this would be to add a map of type `map[uint32]DoltgresType` to `/server/types/base_ids.go` (should probably rename the file to `globals.go` at this point). In `InitBaseIDs()` (should also probably rename that too), we'd add the types by their OID to the map. Then have a function `GetTypeByOID(oid uint32) DoltgresType` that returns the type or `nil` if no match was found.